app/models/restaurant.rb

```diff
@@ -0,0 +1,18 @@
+class Restaurant < ApplicationRecord
+  
+validates :name, presence: true
```

Comment:
your `validates` should be indented once to show that they are "inside" the restaurant class.